libs/common/src/types/mongo-db-models/tfm-facilities.ts

```diff
@@ -66,12 +63,32 @@ export type TfmFacilityAmendment = {
   coveredPercentage?: number;
   requireUkefApproval?: boolean;
   submissionType?: string;
-  submittedByPim?: boolean;
   submittedAt?: UnixTimestamp;
   submissionDate?: UnixTimestamp;
+  effectiveDate?: number;
+  createdBy?: {
+    username: string;
+    name: string;
+    email: string;
+  };
+  leadUnderwriter?: {
+    _id: ObjectId | 'Unassigned';
+    firstName: string;
+    lastName: string;
+  };
+}
+
+/**
+ * Amendments created in TFM
+ *
+ * This type is likely incomplete and should be added
+ * to as and when new properties are discovered
+ */
+export interface TfmFacilityAmendment extends BaseAmendment {
+  type?: typeof AMENDMENT_TYPES.TFM;
```

Comment:
I want to make this a required type but we'll need to migrate amendments in prod - any thoughts?

Reply:
I think we should enforce it, otherwise it will get messy. cc @abhi-markan

Reply:
I think yes but I'll raise a future ticket for this - the PR could bloat quite a lot fixing this otherwise

Reply:
https://ukef-dtfs.atlassian.net/browse/DTFS2-7686

Reply:
Nice one @MarRobSoftwire